### DISCUSSION TITLE
Gray background in multiplayer

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#16948] [Ideas]
In multiplayer, the background now has a gray background. Maybe make it transparent or replace it? He really looks ugly.
(I took a screenshot from one of the pull requests)

![154940207-3166954e-5bf5-4d61-ac01-ab02f09bbb57](https://user-images.githubusercontent.com/59089206/154969936-eb95fd19-5b3f-4724-827e-88b6c43f5113.png)

**Comment by bdach** [comment]
Pretty much none of the newest designs use transparent backgrounds. There is an in-progress redesign of the playlist/multiplayer screens and it also uses a solid background. So I believe changing this would be a no.

If anything a darker shade may end up being used.
